### PR TITLE
rename generator to sandcastle

### LINKED ISSUE #2: we need a new name for this project
so, it's not a generator, it runs commands in a sandbox where the implementation is openshift now (which is possible that it would change in the future)

Ideas:
* sandcastle - funnily enough, there is already a [sandcastle](https://github.com/bcoe/sandcastle) project which is also a sandbox

### PR DESCRIPTION
**Pull request by TomasTomecek**
Fixes #2 

TODO:
* [x] build and test
* [x] fix flake warnings

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,15 +1,15 @@
 .PHONY: build test-build test-in-container push exec-test clean
 
-IMAGE_NAME = docker.io/usercont/packit-generator
-TEST_IMAGE_NAME = docker.io/usercont/packit-generator-test
+IMAGE_NAME = docker.io/usercont/sandcastle
+TEST_IMAGE_NAME = docker.io/usercont/sandcastle
```
Comment (phracek) [inline Makefile]: The same name as for `IMAGE_NAME` as for `TEST_IMAGE_NAME`. I guess typo.

**Comment by phracek** [comment]
I like `sandbox.api` +1

File: Makefile (hunk 1 above)
Comment (TomasTomecek) [reply Makefile]: oh right, good catch!

**Comment by TomasTomecek** [comment]
```
$ pytest-3 -vv
============================================================================ test session starts ============================================================================
platform linux -- Python 3.7.3, pytest-3.9.3, py-1.7.0, pluggy-0.8.1 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /home/tt/g/packit-service/generator, inifile:
plugins: cov-2.6.0
collected 18 items                                                                                                                                                          

tests/e2e/test_ironman.py::test_basic_e2e_inside PASSED                                                                                                               [  5%]
tests/e2e/test_ironman.py::test_run_failure PASSED                                                                                                                    [ 11%]
tests/e2e/test_ironman.py::test_exec_failure PASSED                                                                                                                   [ 16%]
tests/e2e/test_ironman.py::test_local_path_e2e_inside_w_exec PASSED                                                                                                   [ 22%]
tests/e2e/test_ironman.py::test_file_got_changed PASSED                                                                                                               [ 27%]
tests/e2e/test_ironman.py::test_from_pod[test_basic_e2e_inside] PASSED                                                                                                [ 33%]
tests/e2e/test_ironman.py::test_from_pod[test_exec_failure] PASSED                                                                                                    [ 38%]
tests/e2e/test_ironman.py::test_from_pod[test_run_failure] PASSED                                                                                                     [ 44%]
tests/e2e/test_ironman.py::test_from_pod[test_local_path_e2e_inside_w_exec] PASSED                                                                                    [ 50%]
tests/e2e/test_ironman.py::test_from_pod[test_file_got_changed] PASSED                                                                                                [ 55%]
tests/unit/test_pod_and_env.py::test_is_pod_already_deployed PASSED                                                                                                   [ 61%]
tests/unit/test_pod_and_env.py::test_pod_not_deployed PASSED                                                                                                          [ 66%]
tests/unit/test_pod_and_env.py::test_env_image_vars[env_dict0-env_image_vars0] PASSED                                                                                 [ 72%]
tests/unit/test_pod_and_env.py::test_env_image_vars[env_dict1-env_image_vars1] PASSED                                                                                 [ 77%]
tests/unit/test_pod_and_env.py::test_env_image_vars[env_dict2-env_image_vars2] PASSED                                                                                 [ 83%]
tests/unit/test_pod_and_env.py::test_env_image_vars[env_dict3-env_image_vars3] PASSED                                                                                 [ 88%]
tests/unit/test_pod_and_env.py::test_env_image_vars[env_dict4-env_image_vars4] PASSED                                                                                 [ 94%]
tests/unit/test_pod_and_env.py::test_manifest PASSED                                                                                                                  [100%]

======================================================================== 18 passed in 56.89 seconds =========================================================================
```

**Comment by phracek** [comment]
It looks good.

**Comment by phracek** [comment]
I will merge it so that I can use it in https://github.com/packit-service/packit/pull/364

**Comment by phracek** [comment]
Feel free to open new PR if something fails.

**Pull request merged**